__tests__/e2e.test.ts

```diff
@@ -20,14 +20,29 @@ const testRepos: Repo[] = [
   {
     repoPath: 'upleveled/preflight-test-project-react-passing',
     dirName: 'react-passing',
-    // To use install commands, for example to install or upgrade
-    // a package first, pass an installCommands array like this:
-    // installCommands: [
-    //   // To install the latest version of the ESLint config
-    //   'pnpm upgrade --latest @upleveled/eslint-config-upleveled',
-    //   // Avoid any issues with uncommitted files
-    //   'git reset --hard HEAD',
-    // ],
+  },
+  {
+    repoPath: 'upleveled/preflight-test-project-next-js-passing',
```

Comment:
add Next.js project with Postgres.js and Ley:

- https://github.com/upleveled/preflight-test-project-next-js-passing